iree/turbine/kernel/wave/templates/paged_decode_attention.py

```diff
@@ -69,12 +69,12 @@ class Phase(Enum):
         PHASE_1 = (1,)
 
     THREADS_PER_WAVE = 64
-    PHASE_1_BLOCK_B = 64
-    PHASE_1_ELEMS_PER_THREAD = PHASE_1_BLOCK_B // THREADS_PER_WAVE
-    PHASE_1_BLOCK_N = 1
-    HEAD_BLOCK_SIZE = 16
+    PHASE_1_BLOCK_B_WAVES = 1
+    PHASE_1_BLOCK_B = 64 * PHASE_1_BLOCK_B_WAVES
```

Comment:
Is this 64 because there are 64 threads per wave or just a tile size choice?

Reply:
yes, because B is distributed along threads and it need to have at least 1 element_per_thread, I've tried larger `BLOCK_B` to have larger `element_per_thread` but it didn't improve the perf.

Reply:
makes sense, thanks.